compiler/rustc_borrowck/src/borrowck_errors.rs

```diff
@@ -1,9 +1,20 @@
+// #![deny(rustc::untranslatable_diagnostic)]
+// #![deny(rustc::diagnostic_outside_of_impl)]
```

Comment:
If the code can't compile with these uncommented then we can remove the comments until it can :)